app/controllers/welcome_controller.rb

```diff
@@ -0,0 +1,6 @@
+class WelcomeController < ApplicationController
+  def index
+    # # TODO: Jeri, decide if we should keep or delete this
+    # session[:order_id] ||= Order.create(status: "pending").id
```

Comment:
My unsolicited advice: doing this could lead to many orphaned Order records in your database. Do you want to create an Order for every visitor to the site, or just those that begin the cart workflow?